build_containers.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-
 set -euox pipefail
 
+BUILD_ALL=${BUILD_ALL:=0}
+BUILD_NUMBER=${BUILD_NUMBER:=1}
```

Comment:
Is it reasonable to have a default here or should we fail if it's not defined? I was wondering whether that would be something where we'd want to try to avoid conflicts — e.g. maybe using the current Unix timestamp if it's not set:

```suggestion
BUILD_NUMBER=${BUILD_NUMBER:=$(date +%s)}
```

Reply:
This is normally set by Jenkins, but if you try to use the script locally it needs to be something. I'd prefer it to be an integer across the board since it gets used in things like cloudformation resource names that have character limits. I'll put a comment in there.

Reply:
A timestamp will satisfy the integer goal but we could have conflicts if it's started multiple times at the same second which is unlikely except in a CI parallel build scenario.